README.md

```diff
@@ -1,44 +1,43 @@
-<!-- markdownlint-disable-next-line -->
 <div align="center"><p><a href="https://app.netlify.com/sites/devprotocol/deploys"><img src="https://img.shields.io/netlify/bddfebe4-8553-4de6-9ddb-522ce7f67842?style=flat-square&logo=netlify"></a> <a href="https://discord.gg/VwJp4KM"><img src="https://img.shields.io/discord/547215761341546497?style=flat-square&logo=discord"></a> </p><br> <img height="70px" src="https://github.com/web3community/devprotocol.xyz/raw/main/public/assets/logo.png"><br><h1>devprotocol.xyz</h1><h5>Dev Protocol Website 2.0</h5><br><p><img src="https://user-images.githubusercontent.com/91655303/136316732-199891d1-9983-4370-b221-e972bc566d22.png" height="400"></p><p><a href="https://user-images.githubusercontent.com/91655303/136316732-199891d1-9983-4370-b221-e972bc566d22.png"></a> <a href="https://devprotocol.xyz/">View deployed version</a> · <a href="https://github.com/web3community/devprotocol.xyz/issues/new?assignees=&labels=bug&template=bug_report.yml&title=%5BBUG%5D+%3Cdescription%3E">Report a Bug</a></p></div>
 
 ## 🤓 Features
 
-- [x] Uses modern web technologies
-- [x] Source code has support for writing in **Vue**, **React**, **Svelte**, and/or **Preact**
-- [x] Source compiles into static HTML and CSS with no/little JavaScript
-- [x] Uses Tailwindcss for styling
-- [x] Handcrafted UI with UX in mind
-- [x] Accessible, Performant, and SEO Optimized
+*   Uses modern web technologies
+*   Source code has support for writing in **Vue**, **React**, **Svelte**, and/or **Preact**
+*   Source compiles into static HTML and CSS with no/little JavaScript
+*   Uses Tailwindcss for styling
+*   Handcrafted UI with UX in mind
+*   Accessible, Performant, and SEO Optimized
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Accessible, Performant, and SEO Optimized
```